antora-playbook.yml

```diff
@@ -105,13 +105,13 @@ asciidoc:
     prod-operator: che-operator
     prod-operator-image-name: che-operator
     prod-prev-ver-major: "6"
-    prod-prev-ver: "7.30"
+    prod-prev-ver: 7.30
```

Comment:
removing quotes seems like a bad idea... didn't we have a problem with this in the past?

Reply:
@themr0c ? Please, Fabrice, I think you mentioned these quotes already. What is the path we should take from here?

Reply:
Yes, don't remove quotes to version attributes.